include/mega/heartbeats.h

```diff
@@ -37,77 +37,47 @@ namespace mega
 class HeartBeatBackupInfo : public CommandListener
 {
 public:
-    enum Status
-    {
-        UPTODATE = 1, // Up to date: local and remote paths are in sync
-        SYNCING = 2, // The sync engine is working, transfers are in progress
-        PENDING = 3, // The sync engine is working, e.g: scanning local folders
-        INACTIVE = 4, // Sync is not active. A state != ACTIVE should have been sent through '''sp'''
-        UNKNOWN = 5, // Unknown status
-    };
-
-    HeartBeatBackupInfo(int tag, handle aBackupId);
+    HeartBeatBackupInfo(handle backupId);
```

Comment:
I'm missing a virtual dtor here. This class is specialized, so in case of a call `delete` on a pointer of this type, we want dtors of child classes to be called, if any. Better to have them virtual from the beginning, that to forget about it when a specialized class adds something to its dtor., IMHO

Reply:
that would prevents implicit definition of the move constructor and the move assignment operator, but ok, I'll add defaults